code/game/gamemodes/cult/ritual.dm

```diff
@@ -1,240 +1,70 @@
-/obj/item/tome/New()
-	if(!SSticker.mode)
-		icon_state = "tome"
-	else
-		icon_state = SSticker.cultdat.tome_icon
+	force = 15
+	throwforce = 25
+	armour_penetration = 35
+	var/drawing_rune = FALSE
+
+/obj/item/melee/cultblade/dagger/New()
+	if(SSticker.mode)
+		icon_state = SSticker.cultdat.dagger_icon
+		item_state = SSticker.cultdat.dagger_icon
 	..()
 
-/obj/item/tome/examine(mob/user)
+/obj/item/melee/cultblade/dagger/examine(mob/user)
 	. = ..()
 	if(iscultist(user) || user.stat == DEAD)
-		. += "<span class='cult'>The scriptures of [SSticker.cultdat.entity_title3]. Allows the scribing of runes and access to the knowledge archives of the cult of [SSticker.cultdat.entity_name].</span>"
+		. += "<span class='cult'>A dagger gifted by [SSticker.cultdat.entity_title3]. Allows the scribing of runes and access to the knowledge archives of the cult of [SSticker.cultdat.entity_name].</span>"
 		. += "<span class='cult'>Striking another cultist with it will purge holy water from them.</span>"
-		. += "<span class='cult'>Striking a noncultist, however, will sear their flesh.</span>"
+		. += "<span class='cult'>Striking a noncultist, however, will tear their flesh.</span>"
 
-/obj/item/tome/attack(mob/living/M, mob/living/user)
-	if(!istype(M))
-		return
-	if(!iscultist(user))
-		return ..()
+/obj/item/melee/cultblade/dagger/attack(mob/living/M, mob/living/user)
 	if(iscultist(M))
 		if(M.reagents && M.reagents.has_reagent("holywater")) //allows cultists to be rescued from the clutches of ordained religion
 			to_chat(user, "<span class='cult'>You remove the taint from [M].</span>")
 			var/holy2unholy = M.reagents.get_reagent_amount("holywater")
 			M.reagents.del_reagent("holywater")
 			M.reagents.add_reagent("unholywater",holy2unholy)
 			add_attack_logs(user, M, "Hit with [src], removing the holy water from them")
-		return
-	M.take_organ_damage(0, 15) //Used to be a random between 5 and 20
-	playsound(M, 'sound/weapons/sear.ogg', 50, 1)
-	M.visible_message("<span class='danger'>[user] strikes [M] with the arcane tome!</span>", \
-					  "<span class='userdanger'>[user] strikes you with the tome, searing your flesh!</span>")
-	flick("tome_attack", src)
-	user.do_attack_animation(M)
-	add_attack_logs(user, M, "Hit with [src]")
+		return FALSE
+	. = ..()
 
-/obj/item/tome/attack_self(mob/user)
+/obj/item/melee/cultblade/dagger/attack_self(mob/user)
 	if(!iscultist(user))
-		to_chat(user, "<span class='warning'>[src] seems full of unintelligible shapes, scribbles, and notes. Is this some sort of joke?</span>")
+		to_chat(user, "<span class='warning'>[src] is covered in unintelligible shapes and markings.</span>")
 		return
-	open_tome(user)
-
-/obj/item/tome/proc/open_tome(mob/user)
-	var/choice = alert(user,"You open the tome...",,"Scribe Rune","More Information","Cancel")
-	switch(choice)
-		if("More Information")
-			read_tome(user)
-		if("Scribe Rune")
-			scribe_rune(user)
-		if("Cancel")
-			return
-
-/obj/item/tome/proc/read_tome(mob/user)

```

Comment:
actually wish we couldkeep the tome here

Reply:
the tome is only kept for the sake of mapping in cult_items